sdk/keyvault/azure-security-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/SecretAsyncClient.java

```diff
@@ -834,20 +828,19 @@ public String getVaultUrl() {
     }
 
     /**
-     * List all versions of the specified secret. The individual secret response in the flux is represented by {@link
-     * SecretProperties} as only the secret identifier and its attributes are provided in the response. The secret values
-     * are not provided in the response. This operation requires the {@code secrets/list} permission.
+     * Lists all versions of the specified secret. Each {@link SecretProperties secret} returned only has its identifier
+     * and attributes populated. The secret values and secret versions are not listed in the response.
+     * This operation requires the {@code secrets/list} permission.
      *
-     * <p>It is possible to get the Secret with value of all the versions from this information. Convert the {@link
-     * Flux} containing {@link SecretProperties secret} to {@link Flux} containing {@link KeyVaultSecret secret} using
-     * {@link SecretAsyncClient#getSecret(String, String)} within {@link Flux#flatMap(Function)}.</p>
+     * <p><strong>Code sample</strong></p>
+     * <p>The sample below fetches the all the versions of the given secret. For each version retrieved, makes a call
+     * to {@link #getSecret(String, String) getSecret()} to get the version's value, and then prints it out.</p>
```

Comment:
Did you intend to cast the link to getSecret(String, String) like in secret client